shadows/framework/src/main/java/org/robolectric/shadows/ShadowCryptoObject.java

```diff
@@ -11,6 +11,9 @@
 @Implements(value = CryptoObject.class, isInAndroidSdk = false, minSdk = P)
 public class ShadowCryptoObject {
 
+  /** Avoids java.lang.NoSuchMethodError: 'javax.crypto.CipherSpi javax.crypto.Cipher.getCurrentSpi()'
```

Comment:
The Javadoc needs to be clear to tell what this method do, and the reason. You need to describe it with statements not stacktrace only. Other people won't get the purpose of this javadoc. And don't forget java format before you push code.

Reply:
Okay, I think I'll have to let you guys handle this one then as I don't see how I can make this comment better. This method has not been added in the shadow, it is merely a stub. I don't see how describing that it is a stub adds any value.